src/Tools/Installer.php

```diff
@@ -32,11 +32,11 @@ public function install()
         $db->executeQuery('CREATE TABLE `' . Dao::TABLE_NAME . '` (
               `id` int(11) NOT NULL AUTO_INCREMENT,
               `o_id` int(11) NOT NULL,
-              `o_classId` varchar(50) NOT NULL,
+              `classId` varchar(50) NOT NULL,
```

Comment:
we also need a migration for updating existing table

Reply:
Added the migrations for this one